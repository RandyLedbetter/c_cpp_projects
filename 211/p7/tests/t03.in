insert Chico Sacramento 90
insert Chico Redding 60
lookup Chico
lookup Sacramento
lookup Redding
