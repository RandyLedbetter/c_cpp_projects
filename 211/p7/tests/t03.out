Chico: none, Sacramento(90), Redding(60)
Sacramento: Chico(90), none, none
Redding: Chico(60), none, none
